guest-test/tdx/tdx_attest_check.sh

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/bash
+# SPDX-License-Identifier: GPL-2.0-only
+# Copyright (c) 2023 Intel Corporation
+
+# Author:   Hongyu Ning <hongyu.ning@intel.com>
+#
+# History:  16, Oct., 2023 - Hongyu Ning - creation
+
+
+# @desc This script do basic TD attestation check in TDX Guest VM
+#       test binary is based on kselftest linux/tools/testing/selftests/tdx implementation
+
+###################### Variables ######################
+SCRIPT_DIR="$( cd "$( dirname "$0" )" && pwd )"
+echo "$SCRIPT_DIR"
+source common.sh
+
+while getopts :t: arg; do
+	case $arg in
```

Comment:
Please use **2 spaces** for each indentation level instead of tabs in shell files.
It would be great if you could fix the indentation for the whole file.
Great thanks.

Reply:
fixed in new commit

Reply:
> Please use **2 spaces** for each indentation level instead of tabs in shell files. It would be great if you could fix the indentation for the whole file. Great thanks.

@hongyuni Hi Hongyu, as mentioned, please use **2 spaces** for the indentation. @hanning0511 Please correct me if I'm wrong. I remember that we have such rules, right?

Reply:
Yes, refer to https://google.github.io/styleguide/shellguide.html#indentation

Reply:
sorry, updated 2 spaces indentation in latest commit, thought do so but there was some mistake. Would you help to review it?